### PR TITLE
Update respond-machine-alerts.md

### PR DESCRIPTION
**Pull request by msbemba**
Added another note(Disclaimer) about devices that are behind a full VPN tunnel.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4797

File: microsoft-365/security/defender-endpoint/respond-machine-alerts.md [hunk 1]
```diff
@@ -162,6 +162,7 @@ Depending on the severity of the attack and the sensitivity of the device, you m
 >[!IMPORTANT]
 >- Full isolation is available for devices on Windows 10, version 1703.
 >- Selective isolation is available for devices on Windows 10, version 1709 or later.
+>- When isolating a device, only certain processes and destinations are allowed. Therefore, devices that are behind a full VPN tunnel won't be able to reach the Microsoft Defender for Endpoint Cloud service after the device is isolated. The recommendation is to use split-tunneling VPN for Microsoft Defender for Endpoint and Defender AV cloud-based protection related traffic.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/respond-machine-alerts.md]:
```suggestion
>- When isolating a device, only certain processes and destinations are allowed. Therefore, devices that are behind a full VPN tunnel won't be able to reach the Microsoft Defender for Endpoint cloud service after the device is isolated. We recommend using a split-tunneling VPN for Microsoft Defender for Endpoint and Microsoft Defender Antivirus cloud-based protection-related traffic.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@mjcaparas Gentle reminder, please merge this PR. Thanks

**Pull request merged**